module2/src/com/epam/onedimensionalarrays/TaskLauncher.java

```diff
@@ -2,6 +2,22 @@
 
 public class TaskLauncher {
     public static void main(String[] args) {
-        ArraySumm.sumElements();
+        int [] array= {7, -7, -7, 9, -4, 8, 4, 0};
+   //task 1  System.out.println(SumCalculatorOfNumbersDivisibleByK.countSum(array, 2));
+
+  //task 2    System.out.println(CalculatorOfNumberOfReplacements.replaceElementsAndCountReplacements(array, 5));
+
+// task 3  String RealNumb = Arrays.toString(RealNumbersArrayAnalyser.characterizeArray(array,0, 0, 0));
```

Comment:
when I said that you don't need 3 variables to count characteristics, I meant that you don't need to store them as class fields. You just need local variables inside the method, so you don't need to pass them here
```suggestion
   //task 1  System.out.println(SumCalculatorOfNumbersDivisibleByK.countSum(array, 2));

  //task 2    System.out.println(CalculatorOfNumberOfReplacements.replaceElementsAndCountReplacements(array, 5));

// task 3  int[] arrayCharacteristic = RealNumbersArrayAnalyser.characterizeArray(array);
```